app/views/Dashboard/CombinedIndicators/index.tsx

```diff
@@ -4,20 +4,25 @@ import {
     ContainerCard,
     List,
 } from '@the-deep/deep-ui';
-import MultiDataProgressBar from '#components/MultiDataProgressBar';
+import ProgressBar from '#components/ProgressBar';
 import { sourcesProgressBarData } from '#utils/dummyData';
 
 import styles from './styles.css';
 
 const progressBarKeySelector = (d: ProgressBarRendererProps) => d.id;
 
-const barHeight = 15;
+const barHeight = 8;
 export interface ProgressBarRendererProps {
+    className?: string,
+    barHeight?: number,
```

Comment:
remove this from props if not used

Reply:
barHeight is being used in two different component with different height sizes.